src/app/auth/store/auth.reducer.ts

```diff
@@ -0,0 +1,26 @@
+import { createReducer, on } from '@ngrx/store';
+
+import { authSuccess, authError, logoutSuccess } from './auth.actions';
+import { AuthState } from './auth.state';
+
+const initialState: AuthState = {
+  isAuth: false,
+  errors: null
+};
+
+const reducer = createReducer(
+  initialState,
+  on(authSuccess, (state, { isAuth }) => {
+    return { ...state, isAuth, errors: null };
+  }),
+  on(logoutSuccess, (state, { isAuth }) => {
+    return { ...state, isAuth, errors: null };
```

Comment:
I can't really see any reason to path any payload along with `logoutSuccess` action, its name represents everything you need, you can just set `isAuth: false` on this action